f3.go

```diff
@@ -253,21 +240,22 @@ func (m *F3) reconfigure(ctx context.Context, manif *manifest.Manifest) (_err er
 		log.Errorw("failed to properly stop F3 while reconfiguring", "error", err)
 	}
 
-	// pause if new manifest is nil
-	if manif == nil {
-		return nil
-	}
+	// This allows for some possibly unsafe re-configuration
+	m.manifest = manif
+
 	// pause if we are not capable of supporting this version
 	if manif.ProtocolVersion > manifest.VersionCapability {
 		return nil
 	}
+	// Pause if explicitly paused.
+	if manif.Pause {
+		return nil
+	}
 
 	// If we have a new manifest, reconfigure.
-	if m.manifest == nil || m.manifest.NetworkName != manif.NetworkName {
+	if m.manifest.NetworkName != manif.NetworkName {
```

Comment:
I'd move this case up (unset the certificate store if the new manifest name disagrees with the old one.